Alpaca.Markets/Interfaces/IAlpacaDataSubscription.cs

```diff
@@ -12,12 +12,13 @@ public interface IAlpacaDataSubscription
 
     /// <summary>
     /// Gets boolean flag indicating the current subscription status of this item.
+    /// An unsubscribed subscription does not receive streaming data.
     /// </summary>
     [UsedImplicitly]
     Boolean Subscribed { get; }
 
     /// <summary>
-    /// Occurred when a <see cref="Subscribed"/> property value changed.
+    /// Invoked when a <see cref="Subscribed"/> property value has changed.
```

Comment:
I've checked the official .NET documentation ([example](https://docs.microsoft.com/en-us/dotnet/api/system.io.filesystemwatcher.changed?view=net-6.0)) and looks like they prefer to use the `occur` verb for events. But maybe in another tense.